index.php

```diff
@@ -695,7 +695,7 @@ function updateCheckboxAndCookie(layer) {
                 //                                         { layerId      : 1,
                 //                                           wrapDateLine : true
                 //                                         });
-                const osmUrl = OsmTileServer == "BRAVO" ? 'https://t2.openseamap.org/tile/{z}/{x}/{y}.png' : 'https://tile.openstreetmap.org/{z}/{x}/{y}.png';
+                const osmUrl = OsmTileServer !== "BRAVO" ? 'https://t2.openseamap.org/tile/{z}/{x}/{y}.png' : 'https://tile.openstreetmap.org/{z}/{x}/{y}.png';
```

Comment:
This change seems unrelated to the rose, and inverts the sense of the switch for selecting the back-up tileserver.

Reply:
Yes t2 server had CORS issue so I used the other one, but I will put back the old comparaison